### PR TITLE
chore(governance): Update frontmatter insertion to add title case

### PR DESCRIPTION
**Pull request by brandtkeller**
Fulfills consistent page titles for the `governance` webpage as requested in #1257 . More generically ensures all frontmatter for non-index pages is title case across the website. 

Updates developer docs for workflow and rsync dependency.

Some H1's are not fully title case - even with the item in the sidebar having consistent capitalization. Continuing to review.

### CODE REVIEW

**Comment by eddie-knight** [comment]
awkward... the diffs are out of wack with the recent merge in from main.

**Comment by brandtkeller** [comment]
> awkward... the diffs are out of wack with the recent merge in from main.

You are correct... my rebase did not take effectively. Let me get this fixed.

**Comment by eddie-knight** [comment]
After another look... I think the only wonky part is the `presentations.md`, which was deleted in a recent PR?

**Comment by brandtkeller** [comment]
> After another look... I think the only wonky part is the `presentations.md`, which was deleted in a recent PR?

@eddie-knight should be resolved now - contents of this PR reflect intent.

File: website/Makefile [hunk 1]
```diff
@@ -18,7 +18,8 @@ deps:
 		if [ "$$file" = "/_index.md" -o "$$base_name" = "README" ]; then \
 			continue; \
 		fi; \
-		text_to_prepend="---\ntitle: \"$$base_name\"\n---\n"; \
+		title_case=$$(echo "$$base_name" | sed -e 's/-/ /g' -e 's/\b\(.\)/\u\1/g'); \
```
Comment (eddie-knight) [inline website/Makefile:21]: Screenshot 2024-06-20 at 12 55 43 PM

Try this to cover edge cases:

```suggestion
		title_case=$$(echo "$$base_name" | sed -e 's/-/ /g' | awk '{for(i=1;i<=NF;i++) if($$i !~ /^[A-Z]/) $$i=toupper(substr($$i,1,1)) substr($$i,2)}1'); \
```

**Comment by brandtkeller** [comment]
Closing to resolve conflicts -> opening new PR

**Pull request closed without merging**